api/tools/src/main/resources/bigquery/es_person.sql

```diff
@@ -4,9 +4,9 @@ SELECT
   p.gender_concept_id,
   gc.concept_name gender_concept_name,
```

Comment:
Is it needed here too?

Reply:
We currently don't have an unmapped genders in synthetic or prod dbs. But i added it anyway to be safe. Done.